### PR TITLE
Merge Cellpose In To Main 

### PR DESCRIPTION
**Pull request by SRodwin**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 experiment_directory: /gpfs/commons/groups/nygcfaculty/PySeq/20210323_4i4color/pipeline
-output_directory: /gpfs/commons/groups/nygcfaculty/PySeq/20210323_4i4color/pipeline
```
Comment (krpandit) [inline config/config.yaml:2]: We should sanitize this path name and make  it a generic name in user home directory

File: workflow/.tests/unit/common.py
Comment (krpandit) [inline workflow/.tests/unit/common.py:1]: should change 
config_path = '/nethome/kpandit/hudson/workflow/.tests/unit/pytest_config.yaml'
to 
config_path = Path(os.abspath(__file__)).parent / 'pytest_config.yaml'

File: workflow/.tests/unit/test_segmentation.py
Comment (krpandit) [inline workflow/.tests/unit/test_segmentation.py:1]: use test config path fixture for --configfile argument

File: workflow/.tests/unit/test_spatial_neighborhood.py
Comment (krpandit) [inline workflow/.tests/unit/test_spatial_neighborhood.py:1]: use test config path fixture for --configfile argument

**Review by krpandit — CHANGES_REQUESTED** [review]
I put in a few minor comments to change. I also tried running the tests but I got this error below before any tests could actually run. Any ideas why? The tests shouldn't import pytorch, but it should submit a job which does. 


E   ImportError: Failed to load PyTorch C extensions:
E       It appears that PyTorch has loaded the `torch/_C` folder
E       of the PyTorch repository rather than the C extensions which
E       are expected in the `torch._C` namespace. This can occur when
E       using the `install` workflow. e.g.
E           $ python setup.py install && python -c "import torch"
E   
E       This error can generally be solved using the `develop` workflow
E           $ python setup.py develop && python -c "import torch"  # This should succeed
E       or by running Python from a different directory.

File: workflow/.tests/unit/test_spatial_neighborhood.py
Comment (SRodwin) [reply workflow/.tests/unit/test_spatial_neighborhood.py:1]: we no longer run spatial neighborhood

**Pull request merged**